query.go

```diff
@@ -0,0 +1,28 @@
+package scylla
+
+import (
+	"scylla-go-driver/frame"
+	"scylla-go-driver/transport"
+)
+
+type Query struct {
+	stmt transport.Statement
+}
+
+func (q *Query) BindInt64(pos int, v int64) {
+	q.stmt.Values[pos] = frame.Value{
+		N: 8,
+		Bytes: []byte{
+			byte(v >> 56),
+			byte(v >> 48),
+			byte(v >> 40),
+			byte(v >> 32),
+			byte(v >> 24),
+			byte(v >> 16),
+			byte(v >> 8),
+			byte(v),
```

Comment:
Don't we have a helper for that? Note the [ByteOrder](https://pkg.go.dev/encoding/binary).

Reply:
As I look at helper functions from binary package I can see they require slice to have valid length. 
```go
func (bigEndian) PutUint64(b []byte, v uint64) {
	_ = b[7] // early bounds check to guarantee safety of writes below
	b[0] = byte(v >> 56)
	b[1] = byte(v >> 48)
	b[2] = byte(v >> 40)
	b[3] = byte(v >> 32)
	b[4] = byte(v >> 24)
	b[5] = byte(v >> 16)
	b[6] = byte(v >> 8)
	b[7] = byte(v)
}

// PutUvarint encodes a uint64 into buf and returns the number of bytes written.
// If the buffer is too small, PutUvarint will panic.
func PutUvarint(buf []byte, x uint64) int {
	i := 0
	for x >= 0x80 {
		buf[i] = byte(x) | 0x80
		x >>= 7
		i++
	}
	buf[i] = byte(x)
	return i + 1
}
```

I don't like this api, because first of all i want to omit allocation and the second of all I'm not a fan of the flow in which the program creates a slice and then passes it to a function.  It would be nicer if the function could return such slice. 

By creating slice in place (i.e. []byte{byte(x), byte(x >> 8), ...}) I understand we skip allocation, thus I decided to do it this way. 

Please tell me if I don't get it right or I'm missing some function from binary package.